pandas/io/parsers.py

```diff
@@ -2161,6 +2167,52 @@ def _buffered_line(self):
         else:
             return self._next_line()
 
+    def _check_for_bom(self, first_row):
+        """
+        Checks whether the file begins with the BOM character.
+        If it does, remove it. In addition, if there is quoting
+        subsequent to the BOM, remove it as well because it
+        technically takes place at the beginning of the name,
+        not the middle of it.
+        """
+        # first_row will be a list, so we need to check
+        # that that list is not empty before proceeding.
```

Comment:
Indeed, more than I thought.  Currently, tests are failing for Python 2.x because it doesn't see `\ufeff` as a single character.  However, the quickfix of importing `unicode_literals` from `__future__` breaks everything in `parsers`...